src/components/Footer.js

```diff
@@ -7,9 +7,6 @@ export default function Footer({ onClickSave, onClickLoad }) {
     <div className="Footer">
       <Switch>
         <div className="FooterNav">
-          <Route>
-            <h2>CONFIG</h2>
-          </Route>
```

Comment:
I'm just seeing this. Why do you use here `Route`? 
I though we use the `Route` to render some Component, if the path is corresponding to the path in the `Route`
Similiar to this:
```
<Route path="/about">
    <About />
</Route>
```

I think you don't need the `Route` here at all.